ex8/js/ajax.js

```diff
@@ -0,0 +1,18 @@
+function ajax_get(url, callback) {
+  xmlhttp = new XMLHttpRequest();
+  xmlhttp.onreadystatechange = function() {
+    if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
+      console.log('responseText:' + xmlhttp.responseText);
+      try {
+        var data = JSON.parse(xmlhttp.responseText);
```

Comment:
FYI: while this is fine in javascript, it's customary (and mandatory in other languages) to move the `var data` outside of the `try`, because if the `JSON.parse` throws an error, data will remain `undefined`.
So you can do this instead:
`var data = null; // or undefined;
try{
  data = JSON.parse(...)
}
.
.
.
callback(data);`